### DISCUSSION TITLE
Why do `getInputProps` and other helpers need an explicit `ariaDescribedBy` option?

### DISCUSSION THREAD

**Discussion by brunobely** [edmundhung/conform#786] [Q&A]
If the aim of the helpers is to remove boilerplate, why can't it just get it from the field metadata (`metadata.descriptionId`)?

https://github.com/edmundhung/conform/blob/2f2111c376605a4d68a613bd779802622d1ea60b/packages/conform-react/helpers.ts#L157

**Comment by edmundhung** [comment]
It could. But I don't wanna limit cases like multiple elements for description or a custom id setup. This is also why I feel like these helpers are more useful if the implementation are owned by the devs, so they can make changes that fit their requirement best instead of many different options that some people will never use 🤔
